spec/features/user_spec.rb

```diff
@@ -1,12 +1,56 @@
 require 'spec_helper'
 
 feature User do
+  let(:user) { create(:user, name: 'My Name', biography: 'This is my biography!') }
 
-  shared_examples 'admin ability' do
+  describe 'users#show' do
+    let!(:conference) { create(:conference, title: 'My conference') }
+    let!(:unconfirmed_event) { create(:event, state: 'unconfirmed', title: 'This is my test event title!', program: conference.program) }
 
-  end
+    before :each do
+      unconfirmed_event.event_users = [create(:event_user, user: user, event_role: 'submitter')]
+    end
+
+    it 'shows user information' do
+      visit user_path(user)
+      expect(page).to have_content 'My Name'
```

Comment:
And to be honest, I'm not sure if this is really a "feature"...

I'd prefer to test a feature to sign up / sign in and than you can test these on the way ... But just wrapping a view test into a feature test, I don't like it :/

(But just my POV)